components/feature/customtabs/src/main/java/mozilla/components/feature/customtabs/CustomTabsToolbarFeature.kt

```diff
@@ -146,7 +146,10 @@ class CustomTabsToolbarFeature(
         }
 
         val button = Toolbar.ActionButton(drawable, context.getString(R.string.mozac_feature_customtabs_share_link)) {
-            val listener = shareListener ?: { context.share(session.url) }
+            val listener = shareListener ?: {
+                emitActionButtonFact()
```

Comment:
This will emit the fact _only_ if we use the default share sheet and not if there is a custom one. Is that what was intended?

Reply:
Whoops. Good catch. Should be emitted any time--updated!